components/elm/src/main/elmfates_interfaceMod.F90

```diff
@@ -827,8 +842,6 @@ subroutine dynamics_driv(this, bounds_clump, top_as_inst,          &
 
          do ifp = 1, this%fates(nc)%sites(s)%youngest_patch%patchno
             p = ifp+col_pp%pfti(c)
-            this%fates(nc)%bc_in(s)%t_veg24_pa(ifp) = &
```

Comment:
Why is it that we remove this?

Reply:
this was made redundant by https://github.com/NGEET/fates/pull/724

Reply:
yup, we don't even have t_veg24_pa anymore:  https://github.com/NGEET/fates/blob/master/main/FatesInterfaceTypesMod.F90